src/WebJobs.Script.Grpc/WebJobs.Script.Grpc.csproj

```diff
@@ -12,10 +12,17 @@
     </PackageReference>
   </ItemGroup>
 
-  <Target Name="GenerateProtoFiles" BeforeTargets="BeforeBuild">
+  <Target Condition="'$(OS)' == 'Windows_NT'" Name="GenerateProtoFiles" BeforeTargets="BeforeBuild">
```

Comment:
Refactor this to introduce a command name property that is properly set based on the OS, that way we don't need to duplicate the target